packages/wdio-selenium-standalone-service/README.md

```diff
@@ -49,8 +49,8 @@ Control browser driver installation/running separately.
 ```js
 // wdio.conf.js
 const drivers = {
-    chrome: { version: '86.0.4240.22' }, // https://chromedriver.chromium.org/
-    firefox: { version: '0.27.0' }, // https://github.com/mozilla/geckodriver/releases
+    chrome: { version: '91.0.4472.101' }, // https://chromedriver.chromium.org/
+    firefox: { version: '0.29.1' }, // https://github.com/mozilla/geckodriver/releases
```

Comment:
feature request: allow latest

Reply:
Can you please clarify what do you mean? It's possible to set "latest" as version :)

Reply:
no, but it would be nice to just set it as "latest" and either the service or the selenium-standalone package can figure out what to download.

Reply:
https://github.com/vvo/selenium-standalone#available-browsers

https://github.com/vvo/selenium-standalone#available-browsers

Reply:
Ah nice